meta-ros2-jazzy/recipes-bbappends/launch/launch-testing-ament-cmake_3.4.3-1.bbappend

```diff
@@ -0,0 +1,3 @@
+# Copyright (c) 2025 Wind River Systems, Inc.
+
+LICENSE = "BSD-3-Clause"
```

Comment:
[The original .bb file](https://github.com/ros/meta-ros/blob/scarthgap/meta-ros2-jazzy/generated-recipes/launch/launch-testing-ament-cmake_3.4.5-1.bb#L15C1-L16C1) has listed both Apache 2 and BSD. I think the issue is that the text `BSD` is no valid. So, should these changes be:
```
LICENSE = "Apache-2.0 & BSD-3-Clause"
```